Makefile

```
@@ -324,6 +324,7 @@ cluster/prepare/project:
 	@ - oc new-project $(NAMESPACE)
 	@oc label namespace $(NAMESPACE) monitoring-key=middleware --overwrite
 	@oc project $(NAMESPACE)
+	@ - oc label namespace $(NAMESPACE) monitoring-key=middleware openshift.io/cluster-monitoring="true" --overwrite
```

```suggestion
	@oc label namespace $(NAMESPACE) monitoring-key=middleware openshift.io/cluster-monitoring="true" --overwrite
	@oc project $(NAMESPACE)
```
Minor tweak @den-rgb